src/core.c/Main.pm6

```diff
@@ -78,6 +77,11 @@ my sub RUN-MAIN(&main, $mainline, :$in-as-argsfiles) {
               !! coercer(a)
         }
 
+        my @script-options = &main.candidates.map(*.signature.params).flat.map( {
```

Comment:
Truly a minor nitpick, but you have a space between the opening `(` and `{` here, but not between the closing `}` and `)`.